src/webots/nodes/WbCamera.cpp

```diff
@@ -848,6 +848,7 @@ void WbCamera::createWrenCamera() {
   } else {
     mSegmentationCamera = NULL;
     disconnect(mSensor, &WbSensor::stateChanged, this, &WbCamera::updateOverlayMaskTexture);
+    disconnect(mSegmentationCamera, &WbWrenCamera::cameraInitialized, this, &WbCamera::updateSegmentationCameraOrientation);
```

Comment:
We are setting `mSegmentationCamera` to NULL and then we are disconnecting it (NULL).

Reply:
I am not sure why I attached that event, maybe it slipped during some tests.